### PR TITLE
Update readme to document build and serve feature

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: webdev/test/readme_test.dart [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```
Comment (natebosch) [inline webdev/test/readme_test.dart]: 2018

File: webdev/test/test_utils.dart [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```
Comment (natebosch) [inline webdev/test/test_utils.dart]: 2018

File: webdev/README.md [hunk 3]
```diff
@@ -15,19 +15,40 @@ $ pub global activate webdev
...
+```console
+$ webdev help build
+Run builders to build a package.
 
-Global options:
--h, --help    Print this usage information.
+Usage: webdev build [arguments]
+-h, --help            Print this usage information.
+-r, --[no-]release    Build with release mode defaults for builders.
+                      (defaults to on)
```
Comment (natebosch) [inline webdev/README.md]: What do you mean by "on"? 

And where are you setting this default? Was that in another PR?

Comment (natebosch) [reply webdev/README.md]: Ah yeah, found the other PR...

**Comment by kevmoo** [comment]
@natebosch – fixed dates – although I zero epsilon chance it'd cause any issues...

**Comment by kevmoo** [comment]
Nooice!

**Pull request merged**